agent/version/version.go

```diff
@@ -22,10 +22,10 @@ package version
 // repository. Only the 'Version' const should change in checked-in source code
 
 // Version is the version of the Agent
-const Version = "1.55.3"
+const Version = "1.55.4"
 
 // GitDirty indicates the cleanliness of the git repo when this agent was built
 const GitDirty = true
 
 // GitShortHash is the short hash of this agent build
-const GitShortHash = "c0030944"
+const GitShortHash = "983269cb"
```

Comment:
It didn't pick up the right commit. The GitShortHash should be this one [17a7b03e248cb789125ead1e7843fb71e3376d69](https://github.com/aws/amazon-ecs-agent/pull/3047/commits/17a7b03e248cb789125ead1e7843fb71e3376d69) : )